examples/mobile/SchedulingApp/create-event.html

```diff
@@ -17,12 +17,83 @@
 				<a href="#" class="ui-btn ui-btn-icon ui-btn-icon-back" data-style="flat" data-rel="back"></a>
 			</div>
 			<div class="ui-appbar-title-container">
-				<span class="ui-appbar-title">Turn device on</span>
+				<span class="ui-appbar-title">Turn device on and off</span>
 			</div>
 		</header>
 		<div class="ui-content">
-			
+			<div style="font-size: 15px; margin: 0 0 15px 24px">Max running time is 24 hours.
+				<br>Please set time less than 24 hours.
+			</div>
+			<div class="ui-content-subheader">Device on</div>
+			<ul class="ui-listview ui-content-area">
+				<li>
+					<div class="ui-li-text">
+						<!--@ToDo: date+time component -->
+						Sat, Jun20 7:06 AM
+					</div>
+				</li>
+			</ul>
+			<div class="ui-content-subheader">Device off</div>
+			<ul class="ui-listview ui-content-area">
+				<li>
+					<div class="ui-li-text">
+						<!--@ToDo: date+time component -->
+						Sat, Jun20 7:06 AM
+					</div>
+				</li>
+			</ul>
+			<div class="ui-content-subheader">Repeat weekly</div>
+			<ul class="ui-listview ui-content-area">
+				<li>
+					<div class="ui-li-text">
+						<!--@ToDo: day of week component -->
+						<span class="ui-li-text-title">
+							S M T W T F S
+						</span>
+						<span class="ui-li-text-sub">
+							Turn device on setting
+						</span>
+					</div>
+				</li>
+				<li class="ui-li-divider">
+					<div class="ui-li-text">
+						<span class="ui-li-text-title">
+							Mood
+						</span>
+						<span class="ui-li-text-sub ui-li-text-value">
+							Cool
+						</span>
+					</div>
+				</li>
+				<li class="ui-li-divider">
+					<div class="ui-li-text">
+						<span class="ui-li-text-title">
+							Temperature
+						</span>
+						<span class="ui-li-text-sub ui-li-text-value">
+							18°C
+						</span>
+					</div>
+				</li>
+				<li class="ui-li-divider">
```

Comment:
Is it need a divider? It seems a last line.

Reply:
fixed